src/main.lua

```diff
@@ -718,6 +719,7 @@ addEventingHandler(ActionMap.VaultedTransfer, utils.hasMatchingTag("Action", Act
 	local lockLengthMs = msg.Tags["Lock-Length"]
 	local msgId = msg.Id
 	local allowUnsafeAddresses = msg.Tags["Allow-Unsafe-Addresses"] or false
+	local revokable = msg.Tags.Revokable or false
```

Comment:
Be careful with boolean tag parameters. Do they get passed through to here as boolean or as strings? If strings then you'll need a comparison to "true".

Reply:
yes, they also get formatted in assertAndSanitize